server/utils/s3.js

```diff
@@ -112,3 +121,33 @@ export const uploadToS3FromUrl = async (url: string, key: string) => {
     }
   }
 };
+
+export const getSignedImageUrl = async (key: string) => {
```

Comment:
I'm wondering if handling `fake-s3` here would also fix the tests that are proving problematic? 🤔 

> An idea is something similar to using an isDocker Boolean (as in the method above this) to skip generating the signed url and just return ${publicS3Endpoint()}/${key}.

This seems like the right thing to do

Reply:
added the `isDocker` flag, this now solves serving of locally uploaded images in fake-s3 being shown as it is now serving from `http://localhost:4569/bucket/key`.

However it does still require `http://localhost:4569` to allow the upload in the first place. (Seems like this would have always been the case before this branch anyway.)

Reply:
Unfortunately does not fix the redirect tests. I think that is a problem with the test server/library not exposing redirect responses to allow us to assert on. Since the other "first-time" responses of `401` and `403` pass their tests fine.

Reply:
@tommoor think a quick win is to have `app.js` like so:
```javascript
      connectSrc: compact([
        "'self'",
        process.env.AWS_S3_UPLOAD_BUCKET_URL,
        process.env.NODE_ENV !== 'production' && 
       process.env.AWS_S3_UPLOAD_BUCKET_URL.replace(
          's3:',
         'localhost:'
       ),
        'www.google-analytics.com',
      ])
```
with the `_.compact`,  when not needed the element should be removed.

Then, (fingers crossed) that is it for this PR.

Reply:
I'm going to merge and then add that tweak.